### ISSUE TITLE
Images in ch04-assets.tgz are unavaiable.

### ISSUE THREAD

**Issue by after1990s** [qmlbook/qmlbook#51]
All images are write a line "Under Licensing Review".

**Comment by sinadogru** [comment]
Yes same here for me, I would like to have those images, really like to follow this book but without images it just de-motivates me :S 
Btw, thanks for this book really helpful and didactic. Thanks for the efforts on it!

Sina.
